pkg/cmd/dcos_test.go

```diff
@@ -49,3 +52,27 @@ Use "dcos [command] --help" for more information about a command.
 
 	require.Equal(t, expectedHelp, out.String())
 }
+
+func TestPluginEnv(t *testing.T) {
+	cluster := config.NewCluster(nil)
+	cluster.SetURL("https://dcos.example.com")
+	cluster.SetACSToken("abc")
+	cluster.Config().Set("hello.world", "foo")
+	cluster.Config().Set("hallo.world", "foo")
+
+	env := pluginEnv("/path/to/me", "hello", logrus.DebugLevel, cluster)
+
+	require.Contains(t, env, "DCOS_CLI_EXECUTABLE_PATH=/path/to/me")
+	require.Contains(t, env, "DCOS_CLI_VERSION="+version.Version())
+	require.Contains(t, env, "DCOS_VERBOSITY=2")
+	require.Contains(t, env, "DCOS_URL=https://dcos.example.com")
+	require.Contains(t, env, "DCOS_ACS_TOKEN=abc")
+	require.Contains(t, env, "DCOS_HELLO_WORLD=foo")
+	require.NotContains(t, env, "DCOS_HALLO_WORLD=foo")
```

Comment:
I've used ElementsMatch instead of Contains. Regarding `assert` vs `require`, we currently use `require` everywhere on the goroutine running a test. If we want to change this it should be done globally, though assert should be used with care, always making sure the test won't panic afterwards.